src/plugins/unified_doc_viewer/public/components/doc_viewer_logs_overview/logs_overview.tsx

```diff
@@ -10,11 +10,15 @@ import React from 'react';
 import { DocViewRenderProps } from '@kbn/unified-doc-viewer/types';
 import { getLogDocumentOverview } from '@kbn/discover-utils';
 import { EuiHorizontalRule, EuiSpacer } from '@elastic/eui';
+import { ObservabilityLogsAIAssistantFeatureRenderDeps } from '@kbn/discover-shared-plugin/public';
 import { LogsOverviewHeader } from './logs_overview_header';
 import { LogsOverviewHighlights } from './logs_overview_highlights';
 import { FieldActionsProvider } from '../../hooks/use_field_actions';
 import { getUnifiedDocViewerServices } from '../../plugin';
-import { LogsOverviewAIAssistant } from './logs_overview_ai_assistant';
```

Comment:
> [!TIP]
> This file could be removed right?

Reply:
Yes it can, removed here: https://github.com/elastic/kibana/pull/186680/commits/97ba3c1e31c3accd5ed57ec326e3e0dda5440368.